data_node/upload_manager.go

```diff
@@ -95,19 +95,19 @@ func (um *UploadManager) handleInitialUpload(w http.ResponseWriter, r *http.Requ
 	err = createFileDirectory(filepath, 0744)
 	if errors.IsError(err) {
 		log.Println(um.logPrefix, r.RemoteAddr, err)
-		handleRequestError(w, http.StatusBadRequest, "Internal server error")
+		handleRequestError(w, http.StatusServiceUnavailable, "Internal server error")
```

Comment:
Please make the status code and the error message consistent. Let's use InternalServerError status code here.